components/content-blocks/Callout/CalloutEntry/index.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable react/prop-types */
+
+import PropTypes from "prop-types";
+import { useTranslation } from "react-i18next";
+import { makeDateString, useGlobalData } from "@/lib/utils";
+import { Text } from "../styles";
+import {
+  Section,
+  Inner,
+  SharePopup,
+  ImageWrapper,
+  ImageSticker,
+} from "./styles";
+import { Image } from "@rubin-epo/epo-react-lib";
+
+const getDateString = (newsDate, eventStart, eventEnd, lang) => {
+  if (newsDate) {
+    return makeDateString(newsDate, lang);
+  }
+
+  if (eventStart && eventEnd) {
+    return `${makeDateString(eventStart, lang)} - ${makeDateString(
+      eventEnd,
+      lang
+    )}`;
+  }
+
+  if (eventEnd) {
+    return makeDateString(eventEnd, lang);
+  }
+};
+
+export default function CalloutEntry({ callout }) {
+  const { t } = useTranslation();
+  const localeInfo = useGlobalData("localeInfo");
+  const lang = localeInfo?.language || "en-US";
+
+  const { id, entry, backgroundColor } = callout;
+  const {
+    title,
+    date,
+    startDate,
+    endDate,
+    uri,
+    description,
+    image,
+    entryType,
+  } = entry[0];
+  const { title: type, slug: typeSlug } = entryType[0];
+  const titleId = `${typeSlug}-${id}`;
+  const calloutDateString = getDateString(date, startDate, endDate, lang);
+
+  return (
+    <Section $bgColor={backgroundColor} $width="full" $overlay={false}>
+      <Inner href={uri} aria-labelledby={titleId}>
```

Comment:
Using these explicitly in this Callout variant because the Callout `Wrapper` used in the other callouts did not easily allow for customizing the `Section` and `Inner` to the extent needed, and also needed the `SharePopup` to be inside the `Section` but outside the `Inner` to get the clickability and positioning we needed (conversely, `Wrapper` all children get wrapped in `Section` and `Inner`).